controllers/volumereplication_controller.go

```diff
@@ -152,6 +164,28 @@ func (r *VolumeReplicationReconciler) SetupWithManager(mgr ctrl.Manager, cfg *co
 		Complete(r)
 }
 
+// markVolumeAsPrimary defines and runs a set of tasks required to mark a volume as primary
+func (r *VolumeReplicationReconciler) markVolumeAsPrimary(volumeID string, parameters, secrets map[string]string) (string, error) {
+	c := replication.CommonRequestParameters{
+		VolumeID:    volumeID,
+		Parameters:  parameters,
+		Secrets:     secrets,
+		Replication: r.Replication,
+	}
+	var markVolumeAsPrimaryTasks = []*tasks.TaskSpec{
+		{
+			Name: "Enable volume",
```

Comment:
```suggestion
			Name: "Enable volume replication",
```

Reply:
@iamniting can you define some const for these names?